appinventor/components-ios/src/Notifier.swift

```diff
@@ -481,3 +486,15 @@ open class Notifier: NonvisibleComponent {
     EventDispatcher.dispatchEvent(of: self, called: "TextInputCanceled")
   }
 }
+
+func showAlert(message: String?) {
+  if let emessage = message {
+    let alertController = UIAlertController(title: "Error", message: emessage, preferredStyle: .alert)
+    alertController.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
+    UIApplication.shared.keyWindow?.rootViewController?.present(alertController, animated: true, completion: nil)
+  } else {
+    let alertController = UIAlertController(title: "Error", message: "init(coder:) has not been implemented", preferredStyle: .alert)
+    alertController.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
+    UIApplication.shared.keyWindow?.rootViewController?.present(alertController, animated: true, completion: nil)
+  }
+}
```

Comment:
Consider moving this to another file, either its own file in the util group or into Component.swift.